### PR TITLE
tests: add test for nydus with clh

### LINKED ISSUE #4481: nydus: add lazyload support for kata with clh
Add tests support for nydus with clh.

Depends on [kata-containers/kata-containers#3687
](https://github.com/kata-containers/kata-containers/pull/3687)

### PR DESCRIPTION
**Pull request by luodw**
update integration/nydus/nydus_tests.sh to support test nydus with clh

Depends-on: [github.com/kata-containers/kata-containers#3687](https://github.com/kata-containers/kata-containers/pull/3687)

Fixes #4481

Signed-off-by: luodaowen.backend <luodaowen.backend@bytedance.com>

### CODE REVIEW

**Review by jodh-intel — APPROVED** [review]
Thanks @luodw.

lgtm

File: integration/nydus/nydus_tests.sh [hunk 1]
```diff
@@ -53,7 +54,7 @@ function setup_nydus() {
 	sudo -E cp "$dir_path/nydusd-config.json" /etc/
 
 	# start nydus-snapshotter
-	nohup /usr/local/bin/containerd-nydus-grpc \
+        nohup /usr/local/bin/containerd-nydus-grpc \
```
Comment (liubin) [inline integration/nydus/nydus_tests.sh:57]: I'm sorry for this nit: spaces -> tab

Comment (luodw) [reply integration/nydus/nydus_tests.sh]: done

**Review by lifupan — APPROVED** [review]

**Pull request merged**